modules/core/test/test_main.cpp

```diff
@@ -3,4 +3,8 @@
 // of this distribution and at http://opencv.org/license.html.
 #include "test_precomp.hpp"
 
+#if defined(HAVE_HPX) && !defined(HPX_STARTSTOP)
```

Comment:
So, if OpenCV is built without HPX_STARTSTOP, this include shall be put into every single application that uses OpenCV? I believe, it's very big limitation. I would then find some other way to do that.